src/plugins/DevHmrFixInvalidPongPlugin.js

```diff
@@ -1,3 +1,16 @@
+if (!String.prototype.replaceAll) {
+  String.prototype.replaceAll = function(str, newStr){
```

Comment:
Oh gosh, I didn't know that it was added only in Node v15 🤯
I thought it was done somewhere in node 8.

Reply:
yeah its pretty new lol